src/classes/project.py

```diff
@@ -89,12 +89,15 @@ def _load_businesslogic(self):
 
         # Case-sensitive filename we expect
         bl_filename = '{}.xml'.format(self.project_type)
+        bl_file_dir = bl_filename
+        if self.project.attrib['{http://www.w3.org/2001/XMLSchema-instance}noNamespaceSchemaLocation'] == 'https://xml.riverscapes.xyz/Projects/XSD/V2/RiverscapesProject.xsd':
+            bl_file_dir = os.path.join('V2', bl_filename)

```

Comment:
This variable is called `dir` but it has the `name` appended to it. Maybe rename this variable `bl_full_path` or something